### PR TITLE
GRAM-2947 Clone of Fifty / Fifty example app with PHP SDK

### PR DESCRIPTION
**Pull request by bashmach**
## Why?

Example app with usage of new PHP SDK (currently 5.0.0-beta.0)

https://github.com/pipedrive/client-php/blob/5.0.0-beta.0/README.md

## How to test?

Read instructions in README and check demo recording, or follow the next steps:

Required: PHP >= 8.1 (recommended) with sqlite.

```
- run `composer install`
- create a new Marketplace app or use existing one
- copy .env.example to .env and update it with your app properties:
  - PIPEDRIVE_CLIENT_ID=
  - PIPEDRIVE_CLIENT_SECRET=
  - PIPEDRIVE_REDIRECT_URI=http://127.0.0.1:8000/auth/pipedrive/callback
- run `npm run dev` to watch FE assets
- run `php artisan key:generate`
- run `php artisan serve`
```

*NB*: Make sure your app has `deals:full` scope to update deals.

### Expected result

demo

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/86786156-8dee-480d-a934-faf05b385b21)

File: apps/fifty-fifty-php/app/Exceptions/Handler.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        'password_confirmation',
+    ];
+
+    /**
+     * Register the exception handling callbacks for the application.
+     */
+    public function register(): void
+    {
+        $this->reportable(function (Throwable $e) {
+            //
```
Comment (siirimangus) [inline apps/fifty-fifty-php/app/Exceptions/Handler.php:27]: Is the empty comment here needed or could it be removed?

File: apps/fifty-fifty-php/app/Http/Middleware/EncryptCookies.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class EncryptCookies extends Middleware
+{
+    /**
+     * The names of the cookies that should not be encrypted.
+     *
+     * @var array<int, string>
+     */
+    protected $except = [
+        //
```
Comment (siirimangus) [inline apps/fifty-fifty-php/app/Http/Middleware/EncryptCookies.php:15]: The same note about the empty comment. Now I start to think that maybe they're used as some placeholders (?)

File: apps/fifty-fifty-php/app/Http/Middleware/PreventRequestsDuringMaintenance.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+class PreventRequestsDuringMaintenance extends Middleware
+{
+    /**
+     * The URIs that should be reachable while maintenance mode is enabled.
+     *
+     * @var array<int, string>
+     */
+    protected $except = [
+        //
```
Comment (siirimangus) [inline apps/fifty-fifty-php/app/Http/Middleware/PreventRequestsDuringMaintenance.php:15]: The same empty comment question, it cannot be just a coincidence anymore :D

File: apps/fifty-fifty-php/app/Providers/AppServiceProvider.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+use Illuminate\Support\ServiceProvider;
+
+class AppServiceProvider extends ServiceProvider
+{
+    /**
+     * Register any application services.
+     */
+    public function register(): void
+    {
+        //
```
Comment (siirimangus) [inline apps/fifty-fifty-php/app/Providers/AppServiceProvider.php:14]: ok, I'm convinced now that this is meant to be, no more comments about them 😂

**Review by siirimangus — APPROVED** [review]
That's some impressive job 🥇

File: apps/fifty-fifty-php/app/Providers/AppServiceProvider.php (hunk 4 above)
Comment (bashmach) [reply apps/fifty-fifty-php/app/Providers/AppServiceProvider.php:14]: All those empty lines were added by scaffolded laravel app, no special purpose there 😄

**Pull request merged**